symplyphysics/laws/thermodynamics/energy_from_melting.py

```diff
@@ -0,0 +1,35 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output)
+
+# Description
+## The heat of melting is the amount of heat that must be brought to a solid
+# crystalline substance at constant pressure in order to completely transfer it
+# to a liquid state.
+## Q = k_lambda * m
+## Where:
+## Q = energy from melting
```

Comment:
It's not energy from melting. It's energy, required to melt something.
Also rename the law file name.

Reply:
Is the name "energy_to_melt_solid_matter" appropriate?

Reply:
For file name? I think `energy_to_melt_from_mass`

Reply:
> Для имени файла?

Yes